types.go

```diff
@@ -60,3 +60,11 @@ type StringData struct {
 	DataKey
 	Value string
 }
+
+// CustomData is new data type extended by redis module
+// such as Bloom Filter and Cuckoo Filter.
+//
+// CustomData contains only the key of the data.
+type CustomData struct {
```

Comment:
Is this data type necessary?

Reply:
fixed

Reply:
I think it would be better to create new types (e.g. `BloomFilter`, `CuckooFilter`), so users can get more information about the key.

Reply:
fixed https://github.com/tommy351/rdb-go/pull/4/commits/5dba6d5d630103d8393914a0695201bb514ddc11